lib/school_house_web/views/html_helpers.ex

```diff
@@ -14,7 +14,7 @@ defmodule SchoolHouseWeb.HtmlHelpers do
   end
 
   def current_page_locale_path(%{request_path: request_path}, locale) do
-    String.replace(request_path, "/#{current_locale()}/", "/#{locale}/", global: false)
+    String.replace(request_path, "/#{current_locale()}", "/#{locale}", global: false)
```

Comment:
@jaimeiniesta I know you just updated this line, let me know if I need to find a different solution to not cause a regression related to your fix in #86 !

Reply:
@kinson good point on the regression check!

The reason to include that trailing slash is ensuring we're only replacing the locale in its position, and not replace a substring of something that looks like a locale but it's not. Imagine that we add a new section on the site to write about the Essential Elixir Libraries That Every Developer Needs To Know and just like the blog, that URL does not include the locale part, it's just https://beta.elixir.school.com/essential

Then the locale menu would generate again broken links, as this test proves -- you can add it to your branch:

```elixir
test "doesn't break URLs starting with something similar to a locale" do
  assert "/essential" ==
            :get
            |> build_conn("/essential")
            |> HtmlHelpers.current_page_locale_path("ar")
end
```

which fails with:

```
  1) test current_page_locale_path/2 doesn't break URLs starting with something similar to a locale (SchoolHouseWeb.HtmlHelpersTest)
     test/school_house_web/views/html_helpers_test.exs:41
     Assertion with == failed
     code:  assert "/essential" == :get |> build_conn("/essential") |> HtmlHelpers.current_page_locale_path("ar")
     left:  "/essential"
     right: "/arsential"
     stacktrace:
       test/school_house_web/views/html_helpers_test.exs:42: (test)
```

I should have added that test to  protect against this, can you add it to your branch please?

I think we should leave that trailing slash there, if it doesn't break anything else.